server/di-container.ts

```diff
@@ -20,12 +20,28 @@ import {
   validateReplyPUT
 } from './middleware/validation';
 
+// Either use local DB with dev mode on typescript or remote DB in prod mode on server with environment variables
+const path = process.env.ENTITIES || './server/entities/*.ts';
+const connection: any = {
+  type: 'mariadb',
+  url:
+    process.env.JAWSDB_MARIA_URL ||
+    'mysql://BulletinBoarder:fjWvxuY4edDwGiAv6qOa6@localhost:3306/relaa',
+  database: process.env.DATABASE || 'relaa',
+  synchronize: process.env.SYNCHRONIZE || false,
+  logging: process.env.LOGGING || false,
+  entities: [path],
+  cli: {
+    entitiesDir: 'entities'
+  }
+};
+
 const container = awilix.createContainer({
   injectionMode: awilix.InjectionMode.PROXY
 });
 
 export function configureContainer() {
-  return createConnection().then(() => {
+  return createConnection(connection).then(() => {
```

Comment:
Now since database connection stuff is getting pretty big I would move it to own file 👍